suite-native/transactions/src/components/TransactionDetail/TransactionDetailParametersSheet.tsx

```diff
@@ -74,42 +99,56 @@ export const TransactionDetailParametersSheet = ({
                     </TransactionDetailRow>
                 </Card>
 
-                <Card>
-                    {/* TODO: Uncoment when ethereum design is ready. */}
-                    {/* {transaction.symbol === 'eth' && (
-                    <>
-                        <TransactionDetailRow
-                            title="Gas limit"
-                            text={transaction.ethereumSpecific!.gasLimit}
-                        />
-                        <TransactionDetailRow
-                            title="Gas fee"
-                            text={transaction.ethereumSpecific!.gasUsed}
-                        />
-                        <TransactionDetailRow
-                            title="Nonce"
-                            text={transaction.ethereumSpecific!.nonce}
-                        />
-                    </>
-                )} */}
+                {parametersCardIsDisplayed && (
+                    <Card>
+                        {displayedParameters.includes('gasLimit') && (
+                            <TransactionDetailRow title="Gas limit">
+                                {transaction.ethereumSpecific?.gasLimit}
```

Comment:
Shouldn't the condition also include that gas limit can be found on ethereumSpecific property? No need to render if it's undefined so I would add it to the condition as well

```suggestion
                                {transaction.ethereumSpecific.gasLimit}
```

Reply:
Ethereum section reworked in [717bf00](https://github.com/trezor/trezor-suite/pull/7933/commits/717bf00f4d717e813a046e6e4a18555222229742). No optional chaining is used anymore.